plugins/systemair/__init__.py

```diff
@@ -53,6 +55,7 @@ def init_serial_connection(self, serialport, slave_address):
             return False
 
     def _read_modbus(self):
+        self._lockmb.acquire()
```

Comment:
I think the problem is, that a write_modbus triggered by update_item can occure while the read_modbus function is still active. The read_modbus function is called via the scheduler. The if statement in read_modbus prevents the function from running while we have an invalid modbus connection. After 30sec (default value) the serial connection will be established again.